common/contract.ts

```diff
@@ -31,9 +31,10 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   closeTime?: number // When no more trading is allowed
 
   isResolved: boolean
-  resolutionTime?: number // When the contract creator resolved the market
+  resolutionTime?: number // When the market is resolved
   resolution?: string
-
+  autoResolution: resolution
```

Comment:
I don't see why we want this field if auto resolve  always uses one resolution.

If there's another usecase for it, we should add it at that time. I'm also skeptical you would want any specific resolution telegraphed in advanced, other than `N/A`. If we had some api that was auto-resolving it, that should be variable, not fixed in advanced.

Reply:
You are right. I was worried I would need to re-add it if I removed it earlier, but it's worse if we have bloat in our codebase and db. I'll remove it tomorrow.